### ISSUE TITLE
Failed to get chrome version

### ISSUE THREAD

**Issue by zkwabm** [quarto-dev/quarto-cli#4991]
### Bug description

I would like to use Mermaid diagrams in Quarto file and render it to PowerPoint slides, here is a minimal example:

````markdown
---
title: "Untitled"
format: pptx
---

## slides 1
```{mermaid}
graph TD
    A1--> B1
    A2 --> B1
    A3 --> B1
    A4--> B1
    A5 --> B1

```

## slides 2
test
````

Then I got the error below:

```
Error: Failed to get chrome version
    at criClient (file:///D:/App/Quarto/bin/quarto.js:57271:15)
    at async file:///D:/App/Quarto/bin/quarto.js:57239:22
    at async Semaphore.runExclusive (file:///D:/App/Quarto/bin/quarto.js:56923:29)
    at async Object.createPngsFromHtml (file:///D:/App/Quarto/bin/quarto.js:57447:44)
    at async makePng (file:///D:/App/Quarto/bin/quarto.js:83960:72)
    at async makeDefault (file:///D:/App/Quarto/bin/quarto.js:84018:24)
    at async Object.cell (file:///D:/App/Quarto/bin/quarto.js:84041:20)
    at async Promise.all (index 0)
    at async Object.document (file:///D:/App/Quarto/bin/quarto.js:57677:25)
    at async handleLanguageCells (file:///D:/App/Quarto/bin/quarto.js:57648:42)
```

I have already tried to update chrome and quarto to the latest version and tried the command "quarto install tools chromium", but still got the error. If I render the file to html, it works.

And here is Quarto check:

```
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.290
      Path: D:\App\Quarto\bin
      CodePage: 936

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.9.15 (Conda)
      Path: D:/App/Anaconda3/python.EXE
      Jupyter: 5.1.1
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: D:/App/R-4.2.2
      LibPaths:
        - D:/App/R-4.2.2/library
      knitr: 1.42
      rmarkdown: 2.20

[>] Checking Knitr engine render......OK

Quarto version: 1.3.290
Chrome version: Version 111.0.5563.147 (Official Build) (64-bit)
operating system: Win 10
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cderv** [comment]
I can reproduce some issue on Windows chrome (again... 😞 )

**Comment by cderv** [comment]
@zkwabm hopefully next 1.3 release will solve your issue. Please if not solved after trying strictly above v1.3.290. Thanks
